### YamlMime:PythonEnum
uid: azure.mgmt.iothub.v2021_07_02.models.IotHubNameUnavailabilityReason
name: IotHubNameUnavailabilityReason
fullName: azure.mgmt.iothub.v2021_07_02.models.IotHubNameUnavailabilityReason
summary: The reason for unavailability.
module: azure.mgmt.iothub.v2021_07_02.models
fields:
- name: ALREADY_EXISTS
  uid: azure.mgmt.iothub.v2021_07_02.models.IotHubNameUnavailabilityReason.ALREADY_EXISTS
- name: INVALID
  uid: azure.mgmt.iothub.v2021_07_02.models.IotHubNameUnavailabilityReason.INVALID
